.github/workflows/cd.yml

```diff
@@ -24,8 +27,8 @@ jobs:
     strategy:
       matrix:
         arch: [x64, x86]
-        python: ["3.9"]
-        libtorrent: [2.0.7, 1.2.19]
+        python: ["3.7", "3.10"]
```

Comment:
Why do we need 3.7 build?

Reply:
We don't really but I had kept it because it's what deluge was using and/or officially targetting at the moment.

The point here was to have a 3.7 build to represent not breaking the current build, alongside building with newer versions.

Reply:
The actual release only uses one version. There is a difference between CI tests that need to both test oldest and latest support dependencies versus this workflow that creates the Windows release package which should aim to be most stable recent release of dependencies.

Reply:
I just noticed that the packaging test was not running at all on my fork (probably because I never had an event there that would trigger it) so I'll need a bit to check that everything is actually ok on that front.

I'm in the process of looking into some of your other comments and corrections. Also, please take a look at what I said you over on discord.

Reply:
I'm leaving just x64-py3.10 but I put LT1.2 back in due to the outburst in Discord.